cli/README.md

```diff
@@ -9,5 +9,28 @@
 make cli
 ./rill --help 
 ```
+2. To install it via homebrew
+```
+brew install rilldata/rill-developer/rill
+rill --help
+```
+
+3. You can also try our example:
+    1. List of available examples
+    ```
+    rill init list
```

Comment:
Looking at this i dont feel it makes sense to list available examples on `rill init list`. Should there perhaps be `rill init --example list` or something like that?

@hamilton @magorlick Any comments?

Reply:
yeah just added for now, we can update when required. left for the product to test and give any suggestion on this part

Reply:
@AdityaHegde do you mean we should example as subcommand?

Reply:
Updated it to:
```
# creates an empty project
rill init

# creates the default example project
rill init --example

# creates a specific example project
rill init --example NAME

# lists available example projects
rill init --list-examples
```

We can iterate on this during QA, this gets the functionality in there